pylintrc_tweaks

```diff
@@ -5,8 +5,14 @@ init-hook="import sys; sys.path.extend(['lms/djangoapps', 'cms/djangoapps', 'com
 
 [MESSAGES CONTROL]
 # Disable unicode-format-string until we can agree to turn it on.
+# Disable new refactoring suggestions for now, have more important things to fix first
 disable+ =
+    import-outside-toplevel,
+    inconsistent-return-statements,
     native-string,
```

Comment:
Can this be broken into two `disable+ =` lines so that new comment doesn't apply to the older disable?  If not, maybe move `native-string` to the top and add a comment before each section?